### DISCUSSION TITLE
Paddle Festival - Kulturhavn 2024 hos Kulturens Frivillige kan ikke slettes

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#20] [Questions] [answered]
Hvorfor kan jeg ikke slette eventet Paddle Festival - Kulturhavn 2024 hos Kulturens Frivillige? Jeg synes, at jeg har fjernet alle info. Mangler jeg noget?

**Comment by mikkelsl** [comment, marked as answer]
Jeg kan ikke finde eventet, så jeg går ud fra, at det er gået i orden.
